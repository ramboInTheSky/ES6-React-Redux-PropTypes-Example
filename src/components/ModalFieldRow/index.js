import styled from 'styled-components';
import { FieldRow } from '../';

export default styled(FieldRow)`
  clear: both;
  margin-bottom: 0.625rem;
  max-width: 30.9375rem;
`;
