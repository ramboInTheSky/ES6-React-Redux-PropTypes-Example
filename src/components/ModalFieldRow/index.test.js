import React from 'react';
import { shallow } from 'enzyme';

import 'jest-styled-components';

import ModalFieldRow from './';

describe('ModalFieldRow', () => {
  it('should render correctly', () => {
    expect(shallow(<ModalFieldRow />)).toMatchSnapshot();
  });
});
