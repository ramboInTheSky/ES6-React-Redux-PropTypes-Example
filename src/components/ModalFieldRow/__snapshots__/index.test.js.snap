// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFieldRow should render correctly 1`] = `
.c0 {
  clear: both;
  margin-bottom: 0.625rem;
  max-width: 30.9375rem;
}

<styled.div
  className="c0"
/>
`;
